avalon/pipeline.py

```diff
@@ -993,11 +995,11 @@ def load(Loader, representation, namespace=None, name=None, data=None):
                                       "{}".format(Loader.__name__,
                                                   context["subset"]["name"]))
 
-    # Ensure data is a dictionary when no explicit data provided
-    if data is None:
-        data = dict()
+    # Ensure options is a dictionary when no explicit options provided
+    if options is None:
+        options = kwargs.get("data", dict())  # "data" for backward compact
```

Comment:
`backward compact` should be `backward compatibility` or maybe `backward compat` if you really want it shorter? :)